indoor-map/demo.html

```diff
@@ -55,9 +55,10 @@ <h1>Show an Indoor Map</h1>
     <div id="map"></div>
     <h3>Code</h3>
     <p>
-      The example shows loading the Indoor Map using <code>H.venues.Service</code> and renders the indoor map using an instance of <code>H.venues.Provider</code>.<br> 
+      The example shows loading the Indoor Map using <code>H.venues.Service2</code> and renders the indoor map using an instance of <code>H.venues.Provider</code>.<br> 
 	    The example also shows how to:
       <ul>
+        <li>get the list of indoor maps from the given HRN using <code>H.venues.Service2.getMapInfoList()</code></li>
         <li>disable the base map while loading a indoor map using <code>H.Map.setBaseLayer (layer)</code></li>
         <li>switch to an indoor map level other than the default using <code>H.venues.Venue.setActiveLevelIndex (levelIndex)</code></li>
```

Comment:
```suggestion
        <li>switch to an indoor map level other than the default using <code>H.venues.Venue.setActiveLevelIndex(levelIndex)</code></li>
```

Reply:
Done. This is same as the first comment right?